src/Desktop/UIAutomation/DesktopElement.cs

```diff
@@ -101,6 +103,8 @@ public static bool IsFromCurrentProcess(IUIAutomationElement uia)
         /// <returns></returns>
         private static dynamic GetPropertyValue(IUIAutomationElement element, int id)
         {
+            if ( element == null) throw new ArgumentNullException(nameof( element));
```

Comment:
Extra space got introduced in the nameof operation
```suggestion
            if ( element == null) throw new ArgumentNullException(nameof(element));
```